milo/apis/turtleInv.lua

```diff
@@ -0,0 +1,41 @@
+--[[ Emulate a CC:T inventory on the turtle
+]]
+
+local TurtleInv = {}
+
+local turtle = _G.turtle
+
+local inventorySize = 16
+
+local localName = "milo_local_name_unset"
+
+function TurtleInv.setLocalName(name)
+	localName = name
+end
+
+function TurtleInv.size()
+	return inventorySize
+end
+
+function TurtleInv.list()
+	local list = {}
+	for slot = 1, inventorySize do
+		list[slot] = turtle.getItemDetail(slot)
```

Comment:
Only includes the NBT hash with the 'true' second arg.  At least in my testing in 1.18.2; may be different in 1.19

Reply:
Instead of fixing this, I could just wait for cc-tweaked/cc-tweaked#1306 😴